docs/src/tutorials/integrated/changing_snowy_land_parameterizations.jl

```diff
@@ -0,0 +1,219 @@
+# # Changing LandModel Parameterizations
+# In the [Canopy, soil, and snow]() tutorial, we ran the integrated `LandModel`
+# at a Fluxnet site using all of the default parameterizations and parameters.
+
+# In two other tutorials ([Changing soil parameterizations](@ref standalone/Soil/changing_soil_parameterizations.jl)
+# and [Changing canopy parameterizations](@ref standalone/Canopy/changing_canopy_parameterizations.jl))
+# we explored how to change the parameterizations of the standalone soil and canopy models, respectively.
+
+# This tutorial will combine these two streams of work to set up a `LandModel`
+# with non-default parameterizations within the soil and canopy components.
+
+# This time, we'll use non-default parameterizations for two canopy components:
+# - energy model: from the default `BigLeafEnergyModel` to `PrescribedCanopyTempModel`
```

Comment:
in this tutorial you didnt not change this, but I think it's fine! you can remove it and just do the radiation change